src/Compilers/VisualBasic/Portable/Parser/ParseConditional.vb

```diff
@@ -460,6 +472,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return SyntaxFactory.ReferenceDirectiveTrivia(hashToken, referenceKeyword, file)
         End Function
 
+        Private Function ParseShebangDirective(hashToken As PunctuationSyntax) As ShebangDirectiveTriviaSyntax
+            Dim builder = New StringBuilder()
```

Comment:
This has been ported from C# code. Should I then also make the same fix to the C# directive parser?